tests/server/models/plant-test.js

```diff
@@ -0,0 +1,107 @@
+var sinon = require('sinon');
+var expect = require('chai').expect;
+
+var Sequelize = require('sequelize');
+
+var db = require('../../../server/db');
+
+var Plant = db.model('plant');
+
+describe('Plant model', function () {
+
+    beforeEach('Sync DB', function () {
+        return db.sync({ force: true });
+    });
+
+    describe('on creation', function () {
+        it ('has all required fields', function() {
+    	  return Plant.create({
+    	    name: "Rose",
+    	    description: "A plant",
+    	    sun: 2,
+    	    isPerennial: false,
+    	    firstHarvest: 50,
+    	    harvestPeriod: 10,
+    	    afterFrost: true,
+    	    howFarBefore: 0,
+    	    howFarAfter: 14,
+    	    width: 10,
+    	    height: 20
+    		})
+    	    .then(function(product) {
+        	    expect(product.name).to.exist;
+        	    expect(product.description).to.exist;
+        	    expect(product.isPerennial).to.exist;
+        	    expect(product.firstHarvest).to.exist;
+        	    expect(product.harvestPeriod).to.exist;
+        	    expect(product.howFarBefore).to.exist;
+        	    expect(product.howFarAfter).to.exist;
+        	    expect(product.width).to.exist;
+        	    expect(product.height).to.exist;
+    		});
+	    });
```

Comment:
That's almost always because of mixed spaces and tabs. In your editor, tabs show as 2 spaces (for example), whereas on GH they show as 4 spaces. Boom, a mismatch occurs. :-P